<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentQueryDialog</class>
 <widget class="QDialog" name="StudentQueryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生查询</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cb_selection">
       <item>
        <property name="text">
         <string>查询方式 </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按姓名查询</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按学号查询</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按院系查询</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_content"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_search">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
